src/components/AnswersTable/AnswersTable.js

```diff
@@ -0,0 +1,135 @@
+import { html } from '../../shared';
+import styles from './AnswersTable.module.css';
+import check from '../../public/img/check.svg';
+import x from '../../public/img/x.svg';
+
+/**
+ * @typedef {import('../../components/Question').FullAnswerCharacterCategory} FullAnswerCharacterCategory
+ */
+
+/**
+ * @typedef {import('../../components/Question').FullAnswerEpisodeOrLocationCategory} FullAnswerEpisodeOrLocationCategory
+ */
+
+/**
+ * Generates an image element with an icon depending on the correctness of the answer.
+ * @param {'true'|'false'} correct
+ * @returns {HTMLElement}
+ */
+const correctIcon = (correct) => {
+  const img = document.createElement('img');
+  img.className = styles.correctIcon;
+  img.src = correct === 'true' ? check : x;
```

Comment:
```suggestion
 * @param {boolean} correct
 * @returns {HTMLElement}
 */
const correctIcon = (correct) => {
  const img = document.createElement('img');
  img.className = styles.correctIcon;
  img.src = correct ? check : x;
```

IMO nie ma sensu przyjmować tu stringa i zmieniać go z powrotem na boolean, szczególnie, jeżeli w każdym wywołaniu funkcji bierzesz boolean i zamieniasz go w stringa.

Reply:
Chyba z rozpędu do `correctIcon()` wkeiłam `${answer.correct}` zamiast `answer.correct`. Przeoczyłam to a ts krzyczał mi, że string to nie boolean,  stąd taki potwór tutaj 😂